### PR TITLE
Adding RTL support and Arabic lang

### PR DESCRIPTION
**Pull request by NAMEER242**
### What’s New:
- Translated the v1.0.0 content into Arabic.
- Added an option to switch to Arabic.
- Updated the layout to support right-to-left (RTL) for Arabic using a new config field (required for Arabic language).

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -1,38 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="AutoImportSettings">
```
Comment (damianopetrungaro) [inline .idea/workspace.xml:3]: Would you mind dropping `.idea` folder from the repo @NAMEER242 ?

File: themes/conventional-commits/layouts/_default/single.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html class="{{ if .Site.Language.Params.rtl }}rtl{{ end }}">
```
Comment (damianopetrungaro) [inline themes/conventional-commits/layouts/_default/single.html:2]: I think you forgot to handle the top banner section as well:
Screenshot 2024-10-22 at 13 37 42

The text overlaps the background image

File: .idea/workspace.xml (hunk 1 above)
Comment (NAMEER242) [reply .idea/workspace.xml]: Done !!

File: themes/conventional-commits/layouts/_default/single.html (hunk 2 above)
Comment (NAMEER242) [reply themes/conventional-commits/layouts/_default/single.html:2]: According to the Arabic language layout, the title should be on the right, so I made the image stick to the left if the language was Arabic.

Comment (damianopetrungaro) [reply themes/conventional-commits/layouts/_default/single.html:2]: Looks perfect now, thank you @NAMEER242 !
Screenshot 2024-10-23 at 10 56 54

**Review by damianopetrungaro — APPROVED** [review]

**Comment by damianopetrungaro** [comment]
Thank you @NAMEER242 !

**Pull request merged**